practices/advertising.md

```diff
@@ -0,0 +1,54 @@
+# Advertising
+
+These guidelines are based on ethical, [accessibility](accessibility.md) or performance concerns.
+
+
+## Ethical
+
+* Do use clear, intelligible, and unambiguous information.
+* Don't use ads that are misleading. For example, "trick banners" that look like part of the operating system UI.
+* Don't use ads that advertise businesses that participate in spammy or scammy practices.
+* Don't use images that convey a false sense of urgency.
+* Don't advertise products with obvious exaggerations or dubious claims that are impossible to prove.
+* Don't advertise "free" products or services that have hidden costs or charges.
+* Don't advertise gambling, alcohol and tobacco.
+* Don't use [pop-under ads](https://en.wikipedia.org/wiki/Pop-up_ad#Pop-under_ads).
+* Adverts must not contain spyware, malware or any software that results in an unexpected or deceptive experience. This includes links to sites containing these products.
+
+
+## Accessibility
+
+* Don't use ads that make heavy use of fast animations or flashing content.
```

Comment:
Maybe remove the qualifier "heavy" from this sentence so as not to open the door for a discussion around what constitutes heavy use